### PR TITLE
[TEST] Experiment with mouse movements 

### PR DESCRIPTION
**Pull request by ellysmlly**
Hey @adarnoo - really great work so far with your build 

I've start to experiment with the mouse movements that you explained in concept stage to zoom around the page

@sehsarah adding you here too to take a look 

This is a quick video of the vibe at the moment but add any comments explaining what you imagined and we can keep experimenting together

ezgif com-gif-maker

Sweet

### CODE REVIEW

**Comment by adarnoo** [comment]
This is great! Thank you so much for figuring that out.

I'm wondering if its simple to adjust how zoomed in the page in, wanted the images to appear bigger on the screen.

Also, because we made each category on a seperate page I'm wondering if its possible that when you click "Twink" that the words stay on the page and the images appear on top, this being the same for "Bear" and "Jock" I guess I want the words acting as a appear and disappear button.

But this looks great so far! Excited to see it all come together!

**Comment by sehsarah** [comment]
Realised i might need to show you how to 'pull' down changes @adarnoo - can set this up with u on Thurs

**Comment by adarnoo** [comment]
I think I figured it out, it nows works on my GitHub link https://adarnoo.github.io/categories/twink/

I will just need to update my local repo

File: twink/index.html [hunk 1]
```diff
@@ -7,19 +7,24 @@
...
+          <img class="sm-image sm-image--motorcycle" src="../images/twink/motorcycle.gif" alt="motorcycle">
+          <img class="sm-image sm-image--troye" src="../images/twink/troye.jpg" alt="troye">
+          <img class="sm-image sm-image--next-line sm-image--peter" src="../images/twink/peter.jpg" alt="peter">
+          <img class="sm-image sm-image--dick2" src="../images/twink/dick2.gif" alt="dick2">
+          <img class="sm-image sm-image--dick1" src="../images/twink/dick1.gif" alt="dick1">
+          <img class="sm-image sm-image--double-space sm-image--finger" src="../images/twink/finger.jpg" alt="finger">
+          <img class="sm-image sm-image--ass sm-image--next-line" src="../images/twink/ass.jpg" alt="ass">
+          <img class="sm-image sm-image--beachtwink sm-image--five-space" src="../images/twink/beachtwink.png" alt="beachtwink">
+        </div>
+      </div>
```
Comment (ellysmlly) [inline twink/index.html:28]: and if you want to take a look at this mark up and apply the same thing to your other category pages

File: site.css [hunk 2]
```diff
@@ -42,6 +45,24 @@ img {
   opacity: 1;
 }
 
+.sm-image-container--zoom {
+  transform: scale(1.3) translateY(22vw) translateX(10vw);
```
Comment (ellysmlly) [inline site.css:49]: @adarnoo you can play around here to get the zoom the way you intended

**Comment by adarnoo** [comment]
Okay awesome thanks so much!

Quick question, when I am creating new file for javascript in my local folders, am I calling the file custom.js? @ellysmlly

**Comment by sehsarah** [comment]
> I will just need to update my local repo
> when I am creating new file for javascript in my local folders, am I calling the file custom.js?

Yes these 2 describe what i mean about `pull` the changes down. If you're just replicating the changes locally then yes you will need to match exactly to this which is a file called custom.js in the root folder where index.html is

Screen Shot 2021-10-06 at 2 03 16 PM

**Pull request merged**